compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/text/selection/TextFieldSelectionManager.kt

```diff
@@ -1400,3 +1396,5 @@ internal expect fun Modifier.addBasicTextFieldTextContextMenuComponents(
     manager: TextFieldSelectionManager,
     coroutineScope: CoroutineScope,
 ): Modifier
+
+internal expect suspend fun TextFieldSelectionManager.hasAvailableTextToPaste(): Boolean
```

Comment:
Could you please add a todo to upstream? I think this ticket can be added - https://youtrack.jetbrains.com/issue/CMP-7517